src/main.ts

```diff
@@ -83,6 +84,35 @@ export async function activate(
         requiredPackageIds.push('OmniSharp');
     }
 
+    const dotnetRuntimeExtensionId = 'ms-dotnettools.vscode-dotnet-runtime';
+    const requiredDotnetRuntimeExtensionVersion = '2.2.1';
+
+    const dotnetRuntimeExtension = vscode.extensions.getExtension(dotnetRuntimeExtensionId);
+    const dotnetRuntimeExtensionVersion = dotnetRuntimeExtension?.packageJSON.version;
+    if (lt(dotnetRuntimeExtensionVersion, requiredDotnetRuntimeExtensionVersion)) {
+        const button = vscode.l10n.t('Update and reload');
+        const prompt = vscode.l10n.t(
+            'The {0} extension requires version {1} or greater of the .NET Install Tool ({2}) extension.  Please update to continue',
+            context.extension.packageJSON.displayName,
+            requiredDotnetRuntimeExtensionVersion,
+            dotnetRuntimeExtensionId
+        );
+        await vscode.window.showErrorMessage(prompt, button).then(async (selection) => {
```

Comment:
nit: No need to use `.then()`

```suggestion
        const selection = await vscode.window.showErrorMessage(prompt, button);
```